calculator-app/src/Editbox.js

```diff
@@ -0,0 +1,20 @@
+import './App.css';     // import App.css to get the class
+import { useContext  } from "react";   // import useContext 
+import { ButtonContext } from "./App";  // import ButtonContext which was created in the App.js
+
+function Editbox() {
+  const buttoncont = useContext(ButtonContext);   // Define useContext with imported context (ButtonContext)
+  const {digit}=buttoncont;     // destructuring the (buttoncont) context variable which are needed
```

Comment:
Please try like this 
  `const {digit} = useContext(ButtonContext); `
  to avoid `buttoncont` variables.